go-controller/pkg/ovn/controller/services/repair.go

```diff
@@ -97,6 +121,12 @@ func (r *repair) runBeforeSync() {
 	}
 	klog.V(2).Infof("Deleted %d stale service LBs", len(staleLBs))
 
+	// Delete those stale template vars
+	if err := libovsdbops.DeleteChassisTemplateVarByName(r.nbClient, staleTemplateNames.UnsortedList()); err != nil {
```

Comment:
Is there a unit test that exercises this codepath? I looked briefly and could not find one. If there is not, it would be good to add one, agree? It may not have to be part of this PR.

Reply:
I don't think there was any for `runBeforeSync()`.  It might be good to add one in the future indeed.  Shall I open an issue?

Reply:
Yes, good idea. This can be addressed in a separate PR.